configs/containers/docker-ubuntu-clang-mpich.yaml

```diff
@@ -96,7 +101,7 @@ spack:
       os: ubuntu:20.04
       spack:
         url: https://github.com/noaa-emc/spack
-        ref: jcsda_emc_spack_stack
+        ref: release/1.3.0
```

Comment:
This is something we need to do every release. Point the spack ref to the release branch in the container recipes. This particular change will not be merged back to develop.